installing/installing_gcp/installing-gcp-account.adoc

```diff
@@ -26,6 +26,10 @@ include::modules/installation-gcp-service-account.adoc[leveloffset=+1]
 
 include::modules/installation-gcp-permissions.adoc[leveloffset=+2]
 
+include::modules/minimum-required-permissions-gcp-ipi.adoc.adoc[leveloffset=+2]
```

Comment:
```suggestion
include::modules/minimum-required-permissions-gcp-ipi.adoc[leveloffset=+2]
```
looks like you have an extra `.adoc` here.

Reply:
Addressed